### PR TITLE
Cloud Release notes for 25-04-2022 release

### PR DESCRIPTION
**Pull request by sajumb**

### CODE REVIEW

File: Umbraco-Cloud/Release-Notes/2022-04-25-ReleaseNotes.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Release Notes, April 25, 2022
+
+_Enable client certificates from filesystem + Various Cloud Portal improvements_
+
+## Key Takeaways
+- **Enable client certificates from file system** - If you need to load a client certificate from the file system in your cloud project at runtime you can make this possible by turning on the new setting on the _Advanced_ page of the project.
+- **Various tweaks and improvements** - During March and April we have focused on fixing minor issues and adding minor improvements. The highlights of these changes are shown below.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Release-Notes/2022-04-25-ReleaseNotes.md:7]:
```suggestion
- **Various tweaks and improvements** - During March and April, we have focused on fixing minor issues and adding minor improvements. The highlights of these changes are shown below.
```

File: Umbraco-Cloud/Release-Notes/2022-04-25-ReleaseNotes.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## [Enable client certificates from file system](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/Application-Settings/)
+If your cloud project needs to load a client certificate (such as an X.509 certificate) at runtime you can turn on this feature for one or more environments. By turning this feature on for an environment, you will be able to load a client certificate as a file during the run-time of your cloud project.
+
+![Enable client certificate load from file system](images/EnableClientCertificateLoadedFromFileSystem.gif)
+
+## Various tweaks and improvements
+During March and April we have provided a lot of small improvements to the Umbraco Cloud Portal. Actually too many to mention, but you will find the highlights in the list below.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Release-Notes/2022-04-25-ReleaseNotes.md:15]:
```suggestion
During March and April, we have provided a lot of small improvements to the Umbraco Cloud Portal. Actually, too many to mention, but you will find the highlights in the list below.
```

File: Umbraco-Cloud/Release-Notes/2022-04-25-ReleaseNotes.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- Display the time of creation for environments (_now presented on the “Overview” page of the project_)
+- Restrict access to function when environments are changing (_to avoid actions on environments that is being created, deleted or modified_)
```
Comment (eshanrnh) [inline Umbraco-Cloud/Release-Notes/2022-04-25-ReleaseNotes.md:17]:
```suggestion
- Restrict access to function when environments are changing (_to avoid actions on environments that are being created, deleted, or modified_)
```

**Review by eshanrnh — COMMENTED** [review]
Looks good, @sajumb 💪 Just some minor suggestions 🙂

**Comment by sajumb** [comment]
Thanks. All suggestions approved and committed. :-)

**Review by eshanrnh — APPROVED** [review]
Merging it in 🙂

**Pull request merged**